tailcfg/tailcfg.go

```diff
@@ -40,13 +40,13 @@ type CapabilityVersion int
 //
 // History of versions:
 //
-//	 3: implicit compression, keep-alives
-//	 4: opt-in keep-alives via KeepAlive field, opt-in compression via Compress
-//	 5: 2020-10-19, implies IncludeIPv6, delta Peers/UserProfiles, supports MagicDNS
-//	 6: 2020-12-07: means MapResponse.PacketFilter nil means unchanged
-//	 7: 2020-12-15: FilterRule.SrcIPs accepts CIDRs+ranges, doesn't warn about 0.0.0.0/::
-//	 8: 2020-12-19: client can buggily receive IPv6 addresses and routes if beta enabled server-side
-//	 9: 2020-12-30: client doesn't auto-add implicit search domains from peers; only DNSConfig.Domains
+//	3: implicit compression, keep-alives
```

Comment:
is all this reformatting some change in gofmt?

Reply:
Yeah, the new gofmt in 1.19 around auto-formatting of intended bulleted lists gets temperamental about this particular list, probably because it's not a bulleted list like it expects and it's treating it as indented formatted code or something.

I'll send a separate change to make these all into bullets to simplify its job.